package.json

```diff
@@ -19,15 +19,52 @@
     "build:server": "esbuild --packages=external --platform=node --format=esm --outfile=dist/server.js --bundle server/index.ts",
     "test": "vitest",
     "lint": "eslint --ext .js,.jsx,.ts,.tsx .",
-    "format": "prettier -w '**/*.{js,jsx,ts,tsx}'"
+    "format": "prettier -w '**/*.{js,jsx,ts,tsx}'",
+    "storybook": "storybook dev -p 6006",
+    "watch:storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build"
   },
   "eslintConfig": {
-    "extends": "@devacademy/eslint-config/react",
+    "extends": [
```

Comment:
Looks like @devacademy/es-lint/config/react got broken up into an array of letters by mistake